python/makeALPACAEvents.py

```diff
@@ -83,6 +81,12 @@ def inputWrite(mres,gax,nev,Lmin,Lmax):#need to apply Rmin & Rmax as well..
         f.write("****************************************************************************************\n")
         f.write("****************************************************************************************\n")
         f.write("****************************************************************************************\n")
+def getXS(wdir,mres,gax):
```

Comment:
The convention usually is to have blank lines between function definitions.

Reply:
Thank you for your attention.
Done.